fifo.go

```diff
@@ -42,6 +42,12 @@ type fifo struct {
 	handle      *handle
 }
 
+const (
+	F_SETPIPE_SZ = 1031
+)
+
+const FIFO_SIZE = 1024 * 1024
```

Comment:
GitHub actions looks to have purged logs for the previous run, but I think the linter may have been complaining about this, and likely wants them to be combined;

```go
const (
	F_SETPIPE_SZ = 1031
	FIFO_SIZE = 1024 * 1024
)
```

And as both are only used internally, it's better to not export them; perhaps use camelCase to make linters happy, but you can refer to the upstream counterpart for these, e.g.


```go
// Sizes here match what's defined in the kernel; see https://github.com/torvalds/linux/commit/ff9da691c0498ff81fdd014e7a0731dab2337dac
const (
	fsetPipeSize = 1031        // F_SETPIPE_SZ
	fifoSize     = 1024 * 1024 // FIFO_SIZE
)
```

I'm also curios if these are consts that would make sense to be included in `golang.org/x/sys` - this PR is making changes assuming Linux, but the file is used for other unix-y platforms; would this be problematic for any of the other flavours (macOS, freeBSD)? /cc @samuelkarp